apps/versions/models.py

```diff
@@ -48,10 +48,23 @@ def get_query_set(self):
 
 
 def source_upload_path(instance, filename):
+    VALID_SOURCE_EXTENSIONS = (
```

Comment:
This needs to be factorized with https://github.com/mozilla/olympia/blob/master/apps/devhub/forms.py#L359

What I'd do: keep this one, in this file, but move it to the top below the imports, at the module level. Then import it in `apps.devhub.forms`.